metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/security/EnableSecurityConfig.java

```diff
@@ -55,10 +57,10 @@ public void configure(HttpSecurity http) throws Exception {
                 .antMatchers(AUTH_WHITELIST).permitAll()
                 .antMatchers(HttpMethod.GET).permitAll()
                 .antMatchers(HttpMethod.OPTIONS).permitAll()
-                .antMatchers(HttpMethod.POST).hasRole("SERVICE_OPERATOR")
-                .antMatchers(HttpMethod.PUT).hasRole("SERVICE_OPERATOR")
-                .antMatchers(HttpMethod.PATCH).hasRole("SERVICE_OPERATOR")
-                .antMatchers(HttpMethod.DELETE).hasRole("SERVICE_OPERATOR")
+                .antMatchers(HttpMethod.POST).hasRole(ROLE_SERVICE_OPERATOR)
```

Comment:
I guess roles should not contain ROLE_ it should be automatically prefixed by spring.